brainatlas_api/update.py

```diff
@@ -0,0 +1,63 @@
+from rich import print as rprint
+import shutil
+import click
+
+import brainatlas_api
+from brainatlas_api.bg_atlas import _version_str_from_tuple
+
+
+def update_atlas(atlas_name=None, force=False):
+    """
+        Updates a brainatlas_api atlas from the latest
+        available version online. 
+
+        Arguments:
+        ----------
+        atlas_name: str, name of the atlas to update
+        force: bool, if False it checks if the atlas is already
+            at the latest version and doesn't update if
+            that's the case.
+    """
+    # Check input
+    if not isinstance(atlas_name, str):
+        raise ValueError("atlas name should be a string")
+
+    # Get atlas class
+    atlasclass = brainatlas_api.get_atlas_class_from_name(atlas_name)
+    if atlasclass is None:
+        return
+
+    atlas = atlasclass()
+
+    # Check if we need to update
+    if not force:
+        if atlas.check_lateset_version():
+            rprint(
+                f"[b][magenta2]Brainatlas_api: {atlas.atlas_name} is already updated "
+                + f"(version: {_version_str_from_tuple(atlas.local_version)})[/b]"
+            )
+            return
+
+    # Delete atlas folder
+    rprint(
+        f"[b][magenta2]Brainatlas_api: updating {atlas.atlas_name}[/magenta2][/b]"
+    )
+    fld = atlas.brainglobe_dir / atlas.local_full_name
+    shutil.rmtree(fld)
+
+    # Download again
+    atlas.download_extract_file()
+
+    # Check that everything went well
+    atlasclass()
+    rprint(
+        f"[b][magenta2]Brainatlas_api: {atlas.atlas_name} updated to version: "
+        + f"{_version_str_from_tuple(atlas.remote_version)}[/magenta2][/b]"
+    )
+
+
+@click.command()
```

Comment:
These could probably work directly on the update atlas, and be used only when running it from command line

Reply:
it was giving me an error when testing stuff.. not sure if Im doing something stupid with pytest.

Reply:
Ok, then leave it this way and I'll have a look!

Reply:
I had to do the same for the show_atlas, had to make a `cli_show_atlas` otherwise the tests fail...